src/ui/components/ProjectVersion/index.jsx

```diff
@@ -131,6 +134,7 @@ function ProjectVersion() {
       projectInformation.accessToken,
     );
 
+    setPages(responseResult.content);
     if (responseResult.result === "error") {
       setToast({ status: true, message: responseResult.message });
 
```

Comment:
- `P1`
```suggestion
    setPages(responseResult.content);
    
    if (responseResult.result === "error") {
      setToast({ status: true, message: responseResult.message });
```

한줄 개행 부탁드립니다!